zkg

```diff
@@ -37,6 +37,19 @@ if os.path.isdir(ZEEK_PYTHON_DIR):
 else:
     ZEEK_PYTHON_DIR = None
 
+# Similarly, make Zeek's binary installation path available by
+# default. This helps package installations succeed that require
+# e.g. zeek-config for their build process.
+ZEEK_BIN_DIR = '@ZEEK_BIN_DIR@'
+if os.path.isdir(ZEEK_BIN_DIR):
+    try:
+        if ZEEK_BIN_DIR not in os.environ['PATH']:
```

Comment:
That `in` operator would only do a substring-search which may not be correct.

```suggestion
        if ZEEK_BIN_DIR not in os.environ['PATH'].split(os.pathsep):
```

Reply:
Right, that's better. Thanks!